app/routes/repos.py

```
@@ -4,189 +4,192 @@
+	summary="Get all repos",
+	description="Retrieve a paginated list of repositories for a user",
 )
 async def get_repos(
-    user_id: str = Path(
-        ..., description="The ID of the user to retrieve repositories for"
-    ),
-    pagination: PaginationParams = Depends(),
+		user_id: str = Path(
+			..., description="The ID of the user to retrieve repositories for"
+		),
+		pagination: PaginationParams = Depends(),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)